json-format.md

```diff
@@ -70,6 +70,10 @@ with exceptions noted below.
 | URI-reference | [string][json-string] following [RFC 3986][rfc3986]            |
 | Timestamp     | [string][json-string] following [RFC 3339][rfc3339] (ISO 8601) |
 
+Unset attributes MAY be encoded to the JSON value of `null`. When decoding
+attributes and a `null` value is encountered, it MUST be treated as the
+equivilent of unset or omitted.
+
```

Comment:
LGTM
@clemensv  Do we need a similar rule for [AMQP Binary Mode](https://github.com/cloudevents/spec/blob/master/amqp-protocol-binding.md#3132-amqp-application-property-values)? Strictly speaking, `null` values are not allowed there, as `null` in AMQP is a primitive type, and the type mapping does not mention it.

Reply:
IDK, @clemensv might know better, if it is a problem.